security/rspamd/Makefile

```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=          rspamd
+PLUGIN_VERSION=       0.1
+PLUGIN_COMMENT=       Package to configure rspamd to protect your network from malware and spam
+PLUGIN_DEPENDS=       rspamd os-clamav
```

Comment:
still no go on os-clamav

Reply:
why should not I depend on the clamav plugin for av scanning?

Reply:
first, you're depending on another plugin, which is not very elegant and must be avoided. depending on clamav directly would be better (one plugin, one GUI), but already destroys operation of clamav when someone uses the other plugin. so while I see why you this rather than adding clamav directly, the second issue remains:

people will be happy to chose spam and antivirus with two clicks if they want. within the scope of this plugin, we'll concentrate on the spam topic.

Reply:
@fichtner 
The config depends on the clamav plugin to avoid conflicts (the `rspamd` plugin will not write the configuration of clamav because this is already done by another plugin) from this perspective rspamd is a client for the clamd service. If the plugin is not installed, the server will not be added.

I can remove the hard dependency which would make the dependency optional but I would not recommend that.

https://github.com/opnsense/plugins/pull/332/files#diff-ee0bb069a10e58a7287a0b01deef9d6fR24